src/Entity/User.php

```diff
@@ -240,13 +240,24 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, Visibil
     #[Column(type: 'string', nullable: false, options: ['default' => self::USER_TYPE_PERSON])]
     public string $type;
 
+    #[Column(type: 'string', nullable: true)]
+    public string $applicationText;
+
+    #[Column(type: 'boolean', nullable: false, options: ['default' => true])]
+    public bool $isApproved;
+
+    #[Column(type: 'boolean', nullable: false, options: ['default' => false])]
+    public bool $isRejected = false;
```

Comment:
In theory you could potentially end up with `isApproved` and `isRejected` set to true, since there is no guard against this incorrect state.

Not sure how to prevent that.

Reply:
Ah yes of course, this needs to be an enum type. This will also allows you to introduce other states like "waiting" (waiting for approval) safely. 

This is the way.

Reply:
Seems to be fixed now, right? Looks much better with an enum 🥳